google-cloud-tools-plugin/src/com/google/gct/idea/appengine/util/CloudSdkUtil.java

```diff
@@ -83,26 +83,38 @@ public static boolean containsCloudSdkExecutable(@NotNull String path) {
    * @param path @link String} to Cloud SDK binary on local file system.
    * @return a boolean indicating if the file was found.
    */
-  public static boolean isCloudSdkExecutable(@NotNull String path) {
-    VirtualFile vfile = LocalFileSystem.getInstance().findFileByPath(path);
-    return vfile != null && vfile.exists();
+  public static boolean isCloudSdkExecutable(String path) {
+    if (path != null) {
```

Comment:
personally I'd like the failure case to be checked and returned, but just a preference

```
if (path == null) {
  return null
}
```